src/main/java/ch/njol/skript/expressions/ExprItemAmount.java

```diff
@@ -57,8 +56,11 @@ protected String getPropertyName() {
 	}
 	
 	@Override
-	public Number convert(final ItemType item) {
-		return item.getAmount();
+	public Number convert(final Object item) {
+    	if (item instanceof ItemType)
+    		return ((ItemType) item).getAmount();
+    	else
```

Comment:
Don't think we need an else here right? Actually you could probably just use a ternary for this

Reply:
Changed 👍